src/JabbR-Core/Startup.cs

```diff
@@ -43,6 +45,16 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc();
             services.AddSignalR();
+
+            // Establish default settings from appsettings.json
+            services.Configure<ApplicationSettings>(Configuration.GetSection("ApplicationSettings"));
```

Comment:
@adamtuliper Both are required. One reads from the `appsettings.json` file, but the `Version` and `Time` attributes cannot be set statically from the text. Setting version as a raw string causes an exception to be thrown because it can't be implicitly converted to a Version object without a call to `.Parse()`. 

Same goes for the `Time` value, it should be set at runtime so it cannot be read statically from a file.